src/pages/app/posting_project_user/ProfilePage.js

```diff
@@ -24,34 +55,33 @@ import ProjectActivity from "../../../components/app/ProjectInfoPage/ProjectActi
 
 //Firebase stuff
 import { firebase_db } from "../../../firebase-config";
-import { testing_project_id } from "../../../constants/testing-keys";
+import {
+  testing_project_id,
+  testing_organization_id,
+} from "../../../constants/testing-keys";
 
 const imageURL =
   "https://c4.wallpaperflare.com/wallpaper/963/733/213/anime-girls-ghost-blade-wlop-wallpaper-preview.jpg";
 
-const loremText =
-  "Khi Beyond Birthday lấy mạng nạn nhân thứ ba của mình, hắn định làm một thử nghiệm—để xem có cách nào làm một con người chết vì xuất huyết trong mà không cần phải hủy hoại một bộ phận cơ thể nào hay không. Cụ thể, hắn cho nạn nhân chìm vào trạng thái vô thức bằng thuốc mê; ...";
-
 function OrgName(props) {
```

Comment:
Plz push the cmt over here, u can see Ken's code

Reply:
Yea... I know how the comment looks, just don't know how it actually works so I didn't put a comment here, just to avoid misunderstanding.

Reply:
Ok got it